CONTRIBUTING.md

```diff
@@ -1,10 +1,18 @@
 # Přispívání do kódu
 
+* Typické workflow je takové, že si forkneš repository, připravíš úpravy a pak založíš pull request oproti cityvizor repo.
```

Comment:
Chceme po prispevatelich nejakou akci? (comment pod issue "beru si to, dodam za 2 dny/2 hodiny...", join do Slacku?...)

Zaroven, chceme tu pridat sekci "Koho a kde kontaktovat pri problemech/otazkach"?

Reply:
doplněno; koho tam mám hodit jako points of contact?